models/status.go

```diff
@@ -166,6 +167,55 @@ func GetLatestCommitStatus(repo *Repository, sha string, page int) ([]*CommitSta
 	return statuses, x.In("id", ids).Find(&statuses)
 }
 
+// GetLatestCommitStatuses returns all statuses with given repoIDs and shas
+func GetLatestCommitStatuses(repoSHAs []struct {
```

Comment:
Suggestion: change the type of this function's argument to be a slice of issues instead of a slice of these anonymous structs.

I see two main benefits:
1. The code will be cleaner, since we won't need to use this anonymous struct in several different places.
2. The code surrounding the calls to this function inside `repo.Issues(..)` and `home.Issues(..)` are nearly identical. We can move that shared code inside this function by changing the argument type.